osx/QMK Toolbox.xcodeproj/project.pbxproj

```diff
@@ -218,6 +221,43 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		121DE8A222A0E55700A30D70 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\nset -e\nchmod 555 ./dfu-util ./avrdude ./teensy_loader_cli ./dfu-programmer\n";
+		};
+		121DE8A322A0E7E300A30D70 /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\nset -e\nchmod 666 ./dfu-util ./avrdude ./teensy_loader_cli ./dfu-programmer\n";
```

Comment:
If the flashers were moved into a dedicated directory this could look better.

Reply:
Great idea!

Reply:
How about we move all the executables to bin/ ?

Reply:
Sounds good to me.